.github/workflows/gen-docs.yml

```diff
@@ -0,0 +1,105 @@
+name: Generate Documentation
+
+on:
+  release:
+    types: [published]
+    branches: "master"
```

Comment:
We do release from other branches, don't we?

Reply:
We do, but I wanted to get master working first.